rpm/luacheck.spec

```diff
@@ -0,0 +1,41 @@
+Name: luacheck
+Version: 0.0.0
+Release: 1%{?dist}
+Summary: A static analyzer and a linter for Lua
+Group: Development/Tools
+License: MIT
+URL: https://github.com/mpeterv/luacheck
+Source0: luacheck-%{version}.tar.gz
+
+BuildArch: noarch
+
+BuildRequires: tarantool >= 1.7.2.0
+Requires: tarantool >= 1.7.2.0
```

Comment:
Suggested to use 1.9, because it is minimal version we support.

Reply:
Sure.